app/controllers/appoinment_bookings_controller.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+# users service  booking controller
+class AppoinmentBookingsController < ApplicationController
+  def index
+    @opts = if params[:vehicle] == 'two'
+              AppoinmentBooking::Two_wheeler
+            else
+              AppoinmentBooking::Four_wheeler
+            end
+            @bookedappoinment = AppoinmentBooking.where(user_id: params[:id])
+  end
+
+  def new
+    @bookappoinment = AppoinmentBooking.new
+  end
+
+  def create
+    @bookappoinment = AppoinmentBooking.new(allowed_parameter)
+    @bookappoinment.user_id = current_user.id
+    # @bookappoinment.appoinment_booking.build(service_center_id: params[:service_center_id])
+    if @bookappoinment.save
+      flash.alert = 'your appoinment is saved'
+    else
+      flash.alert = 'your appoinment is not saved'
+    end
+    redirect_to root_path
```

Comment:
alert_message = if @bookappoinment.save
                               'your appoinment is saved'
                           else
                              'your appoinment is not saved'
                           end
    redirect_to root_path, a;ert: alert_message